Web.md

```diff
@@ -22,6 +22,73 @@ and retrieve compiled modules from offline storage, instantiate compiled modules
 with JavaScript imports, call the exported functions of instantiated modules,
 alias the exported memory of instantiated modules, etc.
 
+The Web embedding includes additional methods useful in that context.
+Non-web embeddings are not required to support these additional methods.
+
+### Additional Web Embedding API
+
+#### `WebAssembly.compile`
+
+In Web embeddings, the following overload is added (in addition to the core
+JS API method of the same name).
+
+```
+Promise<WebAssembly.Module> compile(Promise<Response> source)
```

Comment:
I'd like to also see this overload to allow:

`Promise<WebAssembly.Module> compile(Response source)`

for symmetry with [`respondWith`](https://w3c.github.io/ServiceWorker/#fetch-event-respondwith)